webapp/components/wpt-runs.js

```diff
@@ -389,6 +389,15 @@ class WPTRuns extends Pluralizer(WPTFlags(SelfNavigation(LoadingState(TestRunsUI
     return link.toString();
   }
 
+  revisionLink(results) {
+    const url = new URL('/results', window.location);
+    url.search = this.query;
+    url.searchParams.set('sha', results.sha);
+    url.searchParams.set('max-count', 1);
```

Comment:
Isn't this the default for /results, so that just omitting does the right thing?

Reply:
Only when no constraints are supplied. Providing a SHA prevents an implicit max-count of 1 (you may want all of the results).

Reply:
That doesn't sound like what I've observed. I've always seen at most one run per product when following links from /runs, has this changed?

Reply:
If there are multiple shas, or a from param, etc, then we don't set max count 1 implicitly.

Reply:
Oh, so some other conditions than what I usually look at, I see.